File: c/array/smallest-element.c
#include <stdio.h>

int main(void)
{
    int arr[10],sm, pos=0;

    printf("Enter 5 elements below:\n");
    for(int i=0; i<5; i++)
    {
        scanf("%d", &arr[i]);
    }

    sm = arr[0];
    for(int i=1; i<5; i++)
    {
        if(arr[i] < sm)
        {
            sm = arr[i];
            pos = i;
        }
    }

    printf("The value of the smallest element is %d.\n", sm);
    printf("The position of the smallest element is %d.", pos);

    return 0;
}

File: java/nopackage/array/SmallestElement.java

import java.util.Scanner;

class SmallestElement {
    public static void main(String[] args)
    {
        int[] arr = new int[10];
        int sm, pos=0;
        Scanner scanner = new Scanner(System.in);

        System.out.printf("Enter 5 elements below:\n");
        for(int i=0; i<5; i++)
        {
            arr[i] = scanner.nextInt();
        }

        sm = arr[0];
        for(int i=1; i<5; i++)
        {
            if(arr[i] < sm)
            {
                sm = arr[i];
                pos = i;
            }
        }

        System.out.printf("The value of the smallest element is %d.\n", sm);
        System.out.printf("The position of the smallest element is %d.", pos);
    }
}